### PR TITLE
Create token-explained.md

### PR DESCRIPTION
**Pull request by Ayushjhax**
Added a detailed token-explained.md file outlining the accounts and tokens in Solana, along with an explanation of the token program. Included images for better visualization and clarity.

### CODE REVIEW

File: content/guides/intro/token-explained.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+## Exploring Different Types of Accounts in Solana
+
+![image](https://github.com/Ayushjhax/developer-content/assets/116433617/cf253051-a943-43d1-a56e-37e8adc39f46)
+
+Now, let's dive into the different types of accounts in Solana, each playing a
+crucial role in the ecosystem.
+
+### Wallet Account
+
+This is like your main wallet where you stash your SOL coins (Solana's own
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:64]: I can do a full PR review later but this won’t pass legal.

File: content/guides/intro/token-explained.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+money). It’s used to derive the accounts for your tokens.
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:65]:
```suggestion
This is your main wallet where you store your SOL (Solana's native token). The address of this account is used to derive the accounts for your tokens.
```

File: content/guides/intro/token-explained.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
+---
+date: Feb 21, 2024
+difficulty: intro
+title: "Tokens Explained"
+description:
+  "Minimal Solana token program skeleton: entry point, parameters for accounts,
+  and instruction data."
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:7]: This discription is out of date too.

File: content/guides/intro/token-explained.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+  blockchain.
+- **Blockchain's answer:** The blockchain finds your balance and sends the
+  answer back to the RPC.
+- **Translating back:** The RPC translates the answer back into English.
+- **Website gets the info:** The RPC sends the translated answer to the website.
+- **You get your answer:** The website shows you your balance in a way you
+  understand!
+
+**Key point:** The Solana RPC makes everything work smoothly, so you don't have
+to worry about learning the complicated language of the blockchain.
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:53]: I would remove this entire section. You're focusing on the different types of accounts here - talking about RPCs isn't necessary.

File: content/guides/intro/token-explained.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+### Wallet Account
+
+This is like your main wallet where you stash your SOL coins (Solana's own
+money). It’s used to derive the accounts for your tokens.
+
+### Token Account
+
+These accounts are associated with users or wallets and hold vital information
+about the tokens. They work in coherence with the wallet account and mint
+account, sharing similar data and intent for performing transactions.
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:71]:
```suggestion
These accounts are associated with wallets and hold a balance of a specific token. Their address is calculated from the wallet account and the mint account for that particular token.
```

File: content/guides/intro/token-explained.md [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+
+### Mint Account
+
+The mint account stores data about the token itself, including properties such
+as mint authority, supply, decimals, and freeze authority. Think of these as the
+instruction manuals for each token. They tell you how many tokens are out there,
+who's allowed to make more, that sort of thing.
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:78]:
```suggestion
### Token Mint Account

The token mint account (sometimes just called a mint account) is like a factory for a specific token. It stores the supply of the token (how many tokens have been created), the mint authority (the account allowed to mint more tokens), the decimals of the token, and (if using Token Extensions Metadata) on-chain metadata about the token, like a name and symbol and URI.
```

Comment (mikemaccana) [reply content/guides/intro/token-explained.md:78]: Much of the information above is already included in your bullet points.

File: content/guides/intro/token-explained.md [hunk 7]
```diff
@@ -0,0 +1,190 @@
...
+
+![image](https://github.com/Ayushjhax/developer-content/assets/116433617/e325ff8a-c020-4ca6-91ad-dda7ea83a32b)
+
+- **Mint authority:** The designation of your own or some other program using
+  which the mint tokens can be signed.
+- **Supply:** Holds information on how many tokens have been issued overall.
+- **Decimals:** The limit until which the token can be broken down into.
+- **Is Initialized:** Flag to see if the account is ready to go (or not).
+- **Freeze authority:** The designation of your own or some other program using
+  which the mint tokens can be frozen.
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:88]:
```suggestion
- **Mint authority:** a wallet or multisig account that is allowed to make (mint) more tokens.
- **Supply:** how many tokens have been issued overall
- **Decimals:** The limit until which the token can be broken down into.
- **Freeze authority:** a wallet or multisig account that is allowed to freeze token accounts.
- **Metadata** (if using Token Extensions Metadata) on-chain metadata about the token, like a name and symbol and URI. 
```

File: content/guides/intro/token-explained.md [hunk 8]
```diff
@@ -0,0 +1,190 @@
...
+labels on your token boxes. They've got the token's name, its ticker symbol
+(like RAD for RadCoin), maybe even a cool picture.
+
+**Example:** Currency has a name (US Dollar), symbol ($ or USD), description,
+and image. Solana’s metadata holds another layer of metadata:
+
+- NAME
+- SYMBOL
+- DESCRIPTION
+- IMAGE
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:103]:
```suggestion
```

We now suggest using token extensions for token metadata.

File: content/guides/intro/token-explained.md [hunk 9]
```diff
@@ -0,0 +1,190 @@
...
+
+## Deep Dive into Tokens
+
+### Creation and Functionality
+
+Tokens in Solana are created using the Token Program, which acts as the Token
+Factory. This program allows users to mint, transfer, and burn tokens between
+accounts, functioning as a factory of accounts that enable token
+functionalities.
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:112]:
```suggestion
Tokens in Solana are created using the Token Program. This program allows users to create token mints, mint tokens, transfer tokens, and burn tokens.
```

Comment (mikemaccana) [reply content/guides/intro/token-explained.md:112]: I've removed the 'factory' analogy as we use this elsewhere (not just in this document) for token mint accounts.

File: content/guides/intro/token-explained.md [hunk 10]
```diff
@@ -0,0 +1,190 @@
...
+and image. Solana’s metadata holds another layer of metadata:
+
+- NAME
+- SYMBOL
+- DESCRIPTION
+- IMAGE
+
+## Deep Dive into Tokens
+
+### Creation and Functionality
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:107]:
```suggestion
## The Token Program and Token Extensions Program
```

File: content/guides/intro/token-explained.md [hunk 11]
```diff
@@ -0,0 +1,190 @@
...
+#### The Solana Token Machine
+
+Solana has a special tool called the Token Program that handles all the token
+stuff. Think of it like a factory:
+
+- **Minting:** This is like printing brand-new tokens based on those instruction
+  manuals (Mint Accounts).
+- **Transferring:** Like moving tokens from your RadCoin box to your friend's.
+- **Burning:** Kind of like tossing tokens in the shredder – they're gone
+  forever!
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:125]:
```suggestion
 - Create new types of token, by making **token mint accounts**
 - Mints for new tokens (if they're a mint authority for the token mint).  
 - Transfer tokens between accounts 
 -  Burn tokens - destroy tokens forever, and get back the storage fees - often done for spam NFTs! 
```

File: content/guides/intro/token-explained.md [hunk 12]
```diff
@@ -0,0 +1,190 @@
...
+## Solana Token Program
+
+This Rust program demonstrates the functionality of a token program in the
+Solana blockchain.
+
+## Overview
+
+The program allows minting, transferring, and burning of tokens based on
+provided instruction data. It interacts with Solana accounts using the Solana
+Program library.
```
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:144]:
```suggestion
```

Comment (mikemaccana) [reply content/guides/intro/token-explained.md:144]: This is redundant.

**Review by mikemaccana — CHANGES_REQUESTED** [review]
Sorry about the wait here and thanks for the contribution! This needs a few tweaks before it can go live. See the suggestions inside.

File: content/guides/intro/token-explained.md (hunk 4 above)
Comment (mikemaccana) [inline content/guides/intro/token-explained.md:53]:
```suggestion
```

**Comment by jacobcreech** [comment]
@mikemaccana @Ayushjhax this feels more like a blog post rather than a guide that should live on Solana.com. Given that docs already exist on token today, I'd recommend self publishing this

**Pull request closed without merging**